LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2016   
+Crown Copyright (c) 2016-2017
```

Comment:
If I recall, it's more usual to do this sort of format:
```
Copyright (c) 2010-15,17 Crown copyright
```
i.e. Something that was worked on from 2010 to 2017 in all years but 2016.